tgui/reload.bat

```diff
@@ -6,3 +6,5 @@ for /f "tokens=3*" %%p in ('REG QUERY "HKCU\Software\Microsoft\Windows\CurrentVe
 )
 REM Copy assets to the BYOND cache
 cmd /c copy assets\* "%DocumentsFolder%\BYOND\cache" /y
```

Comment:
this first directory is just for byond itself, i don't think there's any need to keep this line